### PR TITLE
Upgrade spinner library

### PR DESCRIPTION
**Pull request by alonyb**
This PR is for #10338

Upgrade spinner library to avoid spinner in environments that is not a terminal, like files or IDE's.

Here is PR of spinner library [link](https://github.com/alonyb/spinner/pull/3/files) to check differences
In library we used [go-isatty](https://github.com/mattn/go-isatty), suggested by @medyagh 

I tested in unix systems and windows powershell. It works without problem.

**Before fix**
Screen Shot 2021-02-07 at 23 52 44

**After fix**
Screen Shot 2021-02-07 at 23 48 53

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -105,7 +105,7 @@ require (
 
 replace (
 	git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999
-	github.com/briandowns/spinner => github.com/alonyb/spinner v1.12.1
+	github.com/briandowns/spinner => github.com/alonyb/spinner v1.12.6
```
Comment (afbjorklund) [inline go.mod:108]: You are missing the matching update in go.sum

```diff
-github.com/alonyb/spinner v1.12.1/go.mod h1:QOuQk7x+EaDASo80FEXwlwiA+j/PPIcX3FScO+3/ZPQ=
+github.com/alonyb/spinner v1.12.6/go.mod h1:mQak9GHqbspjC/5iUx3qMlIho8xBS/ppAL/hX5SmPJU=
```

**Review by afbjorklund — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (alonyb) [reply go.mod:108]: thanks @afbjorklund it's done :)

**Review by afbjorklund — APPROVED** [review]

**Comment by afbjorklund** [comment]
Did you make a PR to upstream https://github.com/briandowns/spinner ?

**Comment by alonyb** [comment]
> Did you make a PR to upstream https://github.com/briandowns/spinner ?

@afbjorklund Yep, here is the link if you want to check it [PR](https://github.com/briandowns/spinner/pull/108)

**Pull request merged**